Zune.Net.Inbox/Controllers/MessagingInboxController.cs

```diff
@@ -68,12 +68,13 @@ public ActionResult<MessageDetails> Details(string locale, string zuneTag, strin
         [HttpGet]
         public IActionResult UnreadCont(string locale, string zuneTag)
         {
-            using var ctx = new ZuneNetContext();
-            Member member = ctx.Members.First(m => m.ZuneTag == zuneTag);
-            if (member == null)
-                return StatusCode(StatusCodes.Status400BadRequest, $"User {zuneTag} does not exist.");
+        //     using var ctx = new ZuneNetContext();
```

Comment:
This does not compile right now. Do you have a branch where it does, that I should rebase against?

Reply:
Nope, I just haven't touched messaging in a long time. Don't worry about it